src/bundle/Resources/config/routing.yaml

```diff
@@ -3,3 +3,9 @@ ibexa.search:
     methods: ['GET']
     defaults:
       _controller: 'Ibexa\Bundle\Search\Controller\SearchController::searchAction'
+
+ibexa.search.suggestion:
+    path: /suggestion
+    methods: ['GET']
+    defaults:
+      _controller: 'Ibexa\Bundle\Search\Controller\SuggestionController::suggestAction'
```

Comment:
```suggestion
    controller: 'Ibexa\Bundle\Search\Controller\SuggestionController::suggestAction'
```

Reply:
It's done like that for consistency with the previous record.